src/general/OptionsMenu.cs

```diff
@@ -420,6 +425,9 @@ public override void _Ready()
         guiLightEffectsToggle.RegisterToolTipForControl("guiLightEffects", "options");
         assumeHyperthreading.RegisterToolTipForControl("assumeHyperthreading", "options");
         unsavedProgressWarningEnabled.RegisterToolTipForControl("unsavedProgressWarning", "options");
+
+        // Makes it so that when the viewport changes size it automatically displays
+        GetViewport().Connect("size_changed", this, "DisplayResolution");
```

Comment:
I think `_Notification` might be able to already receive the size changed notification without this signal connection required.

Reply:
Also you need to call the update resolution in _Notification when language changes.